definitions/variable/technology/electricity-expansion.yaml

```diff
@@ -0,0 +1,27 @@
+# List of variables related to the evolution of the electricity system
+# results of capacity expansion models
+
+- New Capacity|Electricity|{Electricity Input}:
```

Comment:
I would rather name this variable as "Capacity Addition".

Reply:
good idea! will do the change